broker/lib/controllers/ServiceFabrikApiController.js

```diff
@@ -296,32 +559,32 @@ class ServiceFabrikApiController extends FabrikBaseController {
       .value();
     return Promise
       .try(() => {
-        if (!backup_guid && !time_stamp) {
-          throw new BadRequest('Invalid input as backup_guid or time_stamp not present');
-        } else if (backup_guid) {
-          return this.validateUuid(backup_guid, 'Backup GUID');
-        } else if (time_stamp) {
-          return this.validateDateString(time_stamp);
+        if (!backupGuid && !timeStamp) {
+          throw new BadRequest('Invalid input as backupGuid or timeStamp not present');
+        } else if (backupGuid) {
+          return this.validateUuid(backupGuid, 'Backup GUID');
+        } else if (timeStamp) {
+          return this.validateDateString(timeStamp);
         }
       })
       .then(() => this.backupStore
-        .getBackupFile(time_stamp ? {
-          time_stamp: time_stamp,
-          tenant_id: tenant_id,
-          instance_id: instance_id,
-          service_id: service_id
+        .getBackupFile(timeStamp ? {
+          time_stamp: timeStamp,
+          tenant_id: tenantId,
+          instance_id: instanceId,
+          service_id: serviceId
         } : {
-          backup_guid: backup_guid,
-          tenant_id: tenant_id
+          backup_guid: backupGuid,
+          tenant_id: tenantId
         })
       )
-      .catchThrow(NotFound, new UnprocessableEntity(`No backup with guid '${backup_guid}' found in this space`))
+      .catchThrow(NotFound, new UnprocessableEntity(`No backup with guid '${backupGuid}' found in this space`))
       .tap(metadata => {
         if (metadata.state !== 'succeeded') {
-          throw new UnprocessableEntity(`Can not restore backup '${backup_guid}' due to state '${metadata.state}'`);
+          throw new UnprocessableEntity(`Can not restore backup '${backupGuid}' due to state '${metadata.state}'`);
         }
         if (!req.manager.isRestorePossible(metadata.plan_id)) {
-          throw new UnprocessableEntity(`Cannot restore backup: '${backup_guid}' to plan:'${metadata.plan_id}'`);
+          throw new UnprocessableEntity(`Cannot restore backup: '${backupGuid}' to plan:'${metadata.plan_id}'`);
         }
       })
       .then(metadata => this.fabrik
```

Comment:
Restore is still coming via CF? Why is this not on the lines of Backup? Backup and restore are one flow. Now is restore going to be in a seperate manager?